endtoend/evaluators/operations.go

```diff
@@ -0,0 +1,231 @@
+package evaluators
+
+import (
+	"context"
+	"fmt"
+
+	ptypes "github.com/gogo/protobuf/types"
+	"github.com/pkg/errors"
+	eth "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	e2e "github.com/prysmaticlabs/prysm/endtoend/params"
+	"github.com/prysmaticlabs/prysm/endtoend/types"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/prysmaticlabs/prysm/shared/testutil"
+	"golang.org/x/exp/rand"
+	"google.golang.org/grpc"
+)
+
+var exitedIndice = params.BeaconConfig().FarFutureEpoch
```

Comment:
Is there a better way to represent this? also naming could be different + have a comment